### PR TITLE
Enable profiling for various components

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref https://github.com/kubernetes/kubernetes/issues/65698

### CODE REVIEW

File: jobs/env/ci-kubernetes-e2e-scalability-common.env [hunk 1]
```diff
@@ -8,7 +8,9 @@ LOGROTATE_MAX_SIZE=5G
 
 # Use IP-aliases for scalability tests.
 KUBE_GCE_ENABLE_IP_ALIASES=true
-# Turn on profiling for scheduler
+# Turn on profiling for various components.
+CONTROLLER_MANAGER_TEST_ARGS=--profiling
```
Comment (shyamjvs) [inline jobs/env/ci-kubernetes-e2e-scalability-common.env]: Just to confirm - are you sure we're not already overriding this for was in high-speed env file?

Comment (wojtek-t) [reply jobs/env/ci-kubernetes-e2e-scalability-common.env]: We do (similarly as we were for scheduler test args).
But let's solve that separately :)

Comment (shyamjvs) [reply jobs/env/ci-kubernetes-e2e-scalability-common.env]: As long as it works, it's ok.

**Pull request merged**